frontend/src/metabase/embedding-sdk/theme/sdk-theme-to-css-vars.ts

```diff
@@ -0,0 +1,78 @@
+import type { FlattenObjectKeys } from "../types/utils";
+
+import type { MetabaseComponentTheme } from "./MetabaseTheme";
+
+type MetabaseComponentThemeKey = FlattenObjectKeys<MetabaseComponentTheme>;
+type CssVariableToThemeMap = Record<string, MetabaseComponentThemeKey>;
+
+export const INTERACTIVE_QUESTION_THEME_OPTIONS = {
```

Comment:
Refactors the theme-to-css-variable mappings into smaller groups, so it's easier to read.